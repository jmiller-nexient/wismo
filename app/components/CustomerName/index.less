@import '../../styles/themes/index.less';

.dte-wismo-customer-name {
    color: @dte-gray-700;
    font-family: @dte-wismo-font-family;
    font-size: @dte-wismo-font-size-1;
    font-style: italic
}