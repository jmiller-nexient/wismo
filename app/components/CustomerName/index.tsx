import * as React from 'react';

import './index.less';

interface ICustomerNameProps {
  value: string;
}

const CustomerName: React.FC<ICustomerNameProps> = (props: ICustomerNameProps) => {
  const { value } = props;

  return (
    <span className="dte-wismo-customer-name">
      {value}
    </span>
  );
};

export default CustomerName;
