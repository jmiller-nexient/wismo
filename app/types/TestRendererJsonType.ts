import { ReactTestRendererJSON } from 'react-test-renderer';

type TestRendererJsonType = ReactTestRendererJSON | null;

export default TestRendererJsonType;
