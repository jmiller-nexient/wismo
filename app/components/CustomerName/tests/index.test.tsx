import React from 'react';
import renderer from 'react-test-renderer';

import TestRendererJsonType from 'types/TestRendererJsonType';

import CustomerName from '..';

describe('<AccountNumber />', () => {
  it('should render correctly', () => {
    const tree: TestRendererJsonType = renderer.create(
      <CustomerName firstName="Test" lastName="Test" />,
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
