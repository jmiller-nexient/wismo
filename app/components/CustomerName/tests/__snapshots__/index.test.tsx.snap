// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AccountNumber /> should render correctly 1`] = `
<span>
  Test Test
</span>
`;
